frontend/src/metabase/entities/users/forms.js

```diff
@@ -7,7 +7,7 @@ import { PLUGIN_ADMIN_USER_FORM_FIELDS } from "metabase/plugins";
 import validate from "metabase/lib/validate";
 import FormGroupsWidget from "metabase/components/form/widgets/FormGroupsWidget";
 
-const NAME_FIELDS = [
+const getNameFields = () => [
```

Comment:
When used as constants, `ttag` is called once and strings do not change when locale changes